demo/src/app.js

```diff
@@ -623,6 +657,28 @@ const AppContent = ({ language, onLanguageClick }) => {
                             </div>
                             <div style={{ flexGrow: 1 }} />
                             <div style={{ alignSelf: 'center' }}>baz</div>
```

Comment:
sx ? Or you don't want to change the div to a box just for that ?

Reply:
I migrated makeStyle/useStyle to emotion but left everything else as is. Can be refactored in a separate PR